src/MatrixSupportView.ts

```diff
@@ -8,4 +8,10 @@ export default class MatrixSupportView extends SupportView {
   constructor(public readonly matrix: INumericalMatrix, $parent: d3.Selection<any>, id: number) {
     super(matrix.coltype, $parent, id);
   }
+
+  protected addDefaultColumn() {
+    console.log('No default column for Matrix');
+
```

Comment:
Or leave the implementation empty and write a comment in it that it should do nothing